### PR TITLE
feature/se-11-create-user-on-dev5

### LINKED ISSUE #57: feature/se-11-create-user-on-dev5
## Problem description
The test user doesn't exist in the dev5, create it for test purposes.

### Reported issues
Loading error: https://github.com/dataquest-dev/dspace-angular/issues/58

### 1. Loading error
I cannot login after I created an user or admin as administrator on the _dev5_, the page just freezes in logging process with loading icon. In the console the FE throws error: 
image

I was trying to reproduce this error on local docker. I just built the dataquest docker images locally and tryied to create user as administrator. But the error doesn't occured, I successfuly logged in as the user. That means there is problem with dev5.

**Temporary solution:** The admin is successfuly added to the dev5 from the github action. I just added command to create a new user after creating a new admin in the _building_scripts/run/start_.

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  3  |    0 |    0 |      0 |         0 |
| Review             |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -   |   -     |        0 |
## Problem description
The test user doesn't exist in the dev5, create it for test purposes.

### Reported issues
Loading error: https://github.com/dataquest-dev/dspace-angular/issues/58

### 1. Loading error
I cannot login after I created an user or admin as administrator on the _dev5_, the page just freezes in logging process with loading icon. In the console the FE throws error: 
image

I was trying to reproduce this error on local docker. I just built the dataquest docker images locally and tryied to create user as administrator. But the error doesn't occured, I successfuly logged in as the user. That means there is problem with dev5.

**Temporary solution:** The admin is successfuly added to the dev5 from the github action. I just added command to create a new user after creating a new admin in the _building_scripts/run/start_.

### CODE REVIEW

**Review by MajoBerger — CHANGES_REQUESTED** [review]
use some normal name for user please. John Doe or something...

File: build-scripts/run/start.bat [hunk 1]
```diff
@@ -11,10 +11,11 @@ call start.frontend.bat nopause
 
 pushd ..\..
 docker-compose --env-file %ENVFILE% -p dq-d7 -f docker/cli.yml run --rm dspace-cli create-administrator -e test@test.edu -f admin -l user -p admin -c en
```
Comment (MajoBerger) [inline build-scripts/run/start.bat]: I propose to change this as well as following line to unify mails. 
e.g. admin@test.edu and user@test.edu

**Review by MajoBerger — CHANGES_REQUESTED** [review]
unify login mails

File: build-scripts/run/start.bat (hunk 1 above)
Comment (milanmajchrak) [reply build-scripts/run/start.bat]: fixed

**Review by MajoBerger — APPROVED** [review]

**Comment by MajoBerger** [comment]
User is created already, this is old and not up to date

**Pull request closed without merging**